app/serializers/workshop_project_click_serializer.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class WorkshopProjectClickSerializer < ActiveModel::Serializer
+  attributes :id, :ip_address, :referrer, :user_agent, :type_of
+  has_one :workshop_project
+  has_one :user
+end
```

Comment:
Why are you creating a serializer for WorkshopProjectClicks if they're never rendered?

Reply:
This was removed in b9c6e368a4e0964d1abf86fe3aba70851497621e